OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -33,6 +33,186 @@ const string Millard2012EquilibriumMuscle::
     STATE_FIBER_LENGTH_NAME = "fiber_length";
 const double MIN_NONZERO_DAMPING_COEFFICIENT = 0.001;
 
+//==============================================================================
+// HELPER FUNCTIONS
+//==============================================================================
+namespace
+{
+
+// Calculates the active fiber force.
+//    @param fiso maximum isometric force
+//    @param a activation
+//    @param fal active-force-length multiplier
+//    @param fv fiber-force-velocity multiplier
+inline double CalcActiveFiberForce(double fiso, double a, double fal, double fv)
```

Comment:
Most other method names in OpenSim use standard camel case, so consider using a lowercase "c" for the start of all the "calc" method names (i.e., `calcActiveFiberForce`).

Reply:
:+1: as noted in CONTRIBUTING.md under Coding standards -> Naming conventions (https://github.com/opensim-org/opensim-core/blob/main/CONTRIBUTING.md#naming-conventions).

Reply:
aaah thats what the  🐫 was about hehe. I see, ty